src/Api.php

```diff
@@ -109,6 +110,16 @@ public function emailForwarder(): EmailForwarder
         return new EmailForwarder($this);
     }
 
+    /**
+     * Return a CNAME (DNS record) instance.
+     *
+     * @return CnameRecord
+     */
+    public function cnameRecord(): CnameRecord
```

Comment:
I think we should have the callable structure as `Dsn()->CnameRecord()->update()`.

Reply:
Do we do that with orders? `Orders->BusinessEmail->create()` ?

Can you provide an example please?

Reply:
We haven't done this to date, but going forward the Api class is going to become huge if we don't address this. I would suggest that we look at refactoring it to this style going forwards. So we have a smaller subset of class per major section i.e. emails, dns and then inside the emails class to have the capabilities there ie add, delete etc.... 

What you've done is fine for now but as it scales upwards, we'll end up with a lot of classes in this file that was my concern.

Reply:
Agree, but to be consistent we would need to refactor the existing work for domains etc which would require a backward incompatible change (and there for a _major_ point change in the package version). https://semver.org/

I think we should leave this for the time being.